.github/workflows/build.yml

```diff
@@ -31,17 +62,41 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      - uses: docker/login-action@v2
+
+      - name: Log into registry
+        if: vars.AZUL_DOCKER_REGISTRY_USER
+        uses: docker/login-action@v3
         with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - uses: docker/build-push-action@v3
+          username: ${{ vars.AZUL_DOCKER_REGISTRY_USER }}
+          password: ${{ secrets.AZUL_DOCKER_REGISTRY_PASSWORD }}
+
+      - name: Build and conditionally push Docker image
+        uses: docker/build-push-action@v5
         with:
+          build-args: |
+            azul_docker_bigquery_emulator_base_image_tag=${{ env.azul_docker_bigquery_emulator_base_image_tag }}
+            azul_docker_bigquery_emulator_internal_version=${{ env.azul_docker_bigquery_emulator_internal_version }}
+            VERSION=${{ env.bigquery_emulator_version }}
           context: .
+          # FIXME: Build arm64 version of azul-bigquery-emulator
+          #        https://github.com/DataBiosphere/azul/issues/6090
+          platforms: linux/amd64
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=local,src=~/.cache/go-build
-          build-args: |
-            VERSION=v${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
+          tags: "${{ format(
+            '{0}{1}:{2}',
+            env.azul_docker_registry,
+            env.azul_docker_bigquery_emulator_image,
+            env.bigquery_emulator_version
+          ) }}"
+          cache-from: |
+            type=local,src=~/.cache/go-build
+            "${{ format(
```

Comment:
I think if you use `|`, you don't have to use `"` or `"` may even become literal, which is not what you'd want.

This is a good site to study: https://yaml-multiline.info/

Reply:
From the build output, it seems like they became literal.